website/index.html

```diff
@@ -21,87 +21,94 @@ <h2>
             </a>
         </header>
         <div class="flex-all-content">
+            <div class="gradient-for-line">
+            </div>
             <div class="line">
                 <div></div>
             </div>
-            <div id="main-content">
+            <div class="main-content">
                 <div class="text-image-flex">
                     <div class="introduction width-mobile">
                         <p>
                             <span>Hi! I am a developer</span>
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             I am a front end and 
-                            back end developer, that creates technical solutions for <br>
+                            back end developer, that creates technical solutions for <br/>
                             companies/clients of all genres.
```

Comment:
Content:
We generally do not use genre as a way to describe a business area in business

Reply:
be78e18